src/picknik_ur_gazebo_scan_and_plan_config/objectives/trajectory_around_hercules.yaml

```diff
@@ -0,0 +1,41 @@
+pose2:
```

Comment:
Can we have `pose1` before `pose2` in this file?

Reply:
Done.